src/conventionalCommits.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 Open Assessment Technologies SA;
+ */
+
+/**
+ * Helper module to work with conventional commits tool
+ */
+const conventionalChangelogCore = require('conventional-changelog-core');
+const conventionalCommitsConfig = require('conventional-changelog-conventionalcommits');
+const conventionalRecommendedBump = require('conventional-recommended-bump');
+const semverParse = require('semver/functions/parse');
+const semverInc = require('semver/functions/inc');
+
+/**
+ * Get the npmPackage
+ *
+ * @param {String} rootDir - the path of the TAO instance root
+ * @param {Boolean} [quiet = true] - if we redirect stdout and stderr to the console
+ * @return {NpmPackage}
+ */
+module.exports = function npmPackageFactory() {
+    return {
```

Comment:
Does this need to be a factory or would the methods work as pure functions exported individually?